java/src/jmri/jmrit/display/CoordinateEdit.java

```diff
@@ -332,7 +332,7 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
     public void initSetLevel() {
         oldX = pl.getDisplayLevel();
         textX = new javax.swing.JLabel();
-        textX.setText("level= " + pl.getDisplayLevel());
+        textX.setText(Bundle.getMessage("Level") + " = " + pl.getDisplayLevel());
```

Comment:
>  I think the MessageFormat class Randall referred to can potentially do much more than simple String concatenation.

Very true.  It can also format the data items it's plugging into the format, etc.  It's like the C printf(..) family in that respect.  But it's fine to use it just to handle string concatenation, because it makes it easier to internationalize strings.  Imagine for example two languages that want to have the items in a different order.  One can use `"abc {0} def {1}"` while another language puts it like `"{1} zyx {0} wvu" without having to put any of that in the code.